test/utils/layout-helpers-test.ts

```diff
@@ -206,4 +207,30 @@ describe('definition-helpers', function () {
       await info.destroy();
     });
   });
+
+  describe('isRootStartingWithFilePath', function () {
+    it('should return true if the root path exactly matches the file path', function () {
+      const rootPath = 'foo/bar/biz';
+      const filePath = 'foo/bar/biz/lib/boo.js';
+      const doesStartWithRootPath = isRootStartingWithFilePath(rootPath, filePath);
+
+      expect(doesStartWithRootPath).toBe(true);
+    });
+
+    it('should return false if the root path partially matches the file path', function () {
+      const rootPath = 'foo/bar/biz';
+      const filePath = 'foo/bar/biz-blah/lib/boo.js';
+      const doesStartWithRootPath = isRootStartingWithFilePath(rootPath, filePath);
+
+      expect(doesStartWithRootPath).toBe(false);
+    });
+
+    it('should return false if the root path partially matches the file path', function () {
```

Comment:
could we have windows-like test too?
like 
`c:\\my-folder\my-path\my-file.ts`

Reply:
done